src/test/java/seedu/address/logic/commands/AddNotesCommandTest.java

```diff
@@ -2,35 +2,96 @@
 
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
-import static seedu.address.logic.commands.AddNotesCommand.MESSAGE_ARGUMENTS;
 import static seedu.address.logic.commands.CommandTestUtil.VALID_NOTES_AMY;
 import static seedu.address.logic.commands.CommandTestUtil.VALID_NOTES_BOB;
 import static seedu.address.logic.commands.CommandTestUtil.assertCommandFailure;
+//import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
```

Comment:
Can I assume these lines are commented out for now because this test class has not finished implementation yet?